src/main/java/org/folio/helper/RestoreHelper.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.helper;
+
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import org.folio.rest.core.models.RequestContext;
+import org.folio.rest.jaxrs.model.Piece;
+import org.folio.rest.jaxrs.model.ReceivingCollection;
+import org.folio.rest.jaxrs.model.ReceivingResults;
+
+import java.util.Map;
+
```

Comment:
lets add java doc that this class is responsible to change staus from Unreciavable to Expected, because without context it is hard to understand